### DISCUSSION TITLE
how to stop simulation from generating .sca files

### DISCUSSION THREAD

**Discussion by kevenjxb** [omnetpp/omnetpp#1242] [Q&A]
I'm using omnet++ to train a dqn for flow control, and I dont need the result files of the training episodes. I tried 
**.statistic-recording = false
**.scalar-recording = false
**.vector-recording = false
in my ini file, and it doesnt output .vec files. But .sca files are still generated with only iteration variables and run attributes inside. The manual said when scalar-recording is specified globally to turn off all scalars, no output scalar file (.sca) will be created either. But actually not, please tell me how to fix this

**Comment by avarga** [comment]
This is a bug. Created an issue from it: https://github.com/omnetpp/omnetpp/issues/1244
